Making Glock accept silencer and autolathe emaggable to build silencers

Limeliz opened the pull request:
Made the silencer fit on the Glock (Glock has new typepath).
Made the Autolathe emaggable.
Emagged Autolathe can now build silencers (costs 200k metal, which means the Autolathe needs to be upgraded some).
New sprites for Glock so it can fit the silencer:
![New Glock sprites](http://i.imgur.com/Gk8kQpu.png)

Fixes #855

Limeliz committed:
Adding silencer to emagged autolathe

Made autolathe emaggable.
Added silencer to autolathe (available for 200k metal when emagged).

Limeliz committed:
Made the silencer fit on the Glock

Silencer now fits on the Glock.
Silencer costs 200k metal to produce from an emagged Autolathe.

Limeliz committed:
Removed commented out code

Forgot to remove some commented out code that was never needed.

Limeliz committed:
Added silenced Glock

New sprites for Glock, including two with silencer attached.

ike709 commented on code/game/machinery/autolathe.dm:
```diff
@@ -151,6 +157,15 @@ var/global/list/autolathe_recipes_hidden = list( \
+		var/datum/effect/effect/system/spark_spread/s = new /datum/effect/effect/system/spark_spread
+		s.set_up(5, 1, src)
+		s.start()
+		user << "You short out the [src]'s circuit board, enabling debug mode."
```
Add a span class please. Preferably 'warning' but 'notice' works.

ike709 commented on code/game/machinery/autolathe.dm:
```diff
@@ -151,6 +157,15 @@ var/global/list/autolathe_recipes_hidden = list( \
+		s.set_up(5, 1, src)
+		s.start()
+		user << "You short out the [src]'s circuit board, enabling debug mode."
+		visible_message("\red BZZzZZzZZzZT")
```
You should never use \red, \blue, or \any color. Instead use span class.

ike709 commented:
Also, you have an error. You change the type path for the glock but didn't change it every time it is used in the code. In dream maker use Ctrl + F to search for the old type path(s) and change them to the new type path. You could use Find and Replace but there shouldn't be more than one or two type paths to correct. Otherwise this PR is great.

Limeliz committed:
SPAN CLASSES

Limeliz committed:
Silencer summoned with Glock when Summon Weapons.

Added silencer to be summoned together with the Glock during Summon Weapons.
Updated Glock typepath.

Limeliz commented:
@ike709 Thanks, forgot about span classes (fixed by 29e3c0c). Also forgot to upload summon_weapons.dm to git (fixed by 26dcb62).

Limeliz commented:
This is finished and is ready to be merged.

Bawhoppen commented:
I know I'm kinda late to the party... but I still like this PR and wanna see it get merged... It deserves it.
